src/Eppie.CLI/Eppie.CLI/Resources/Program.resx

```diff
@@ -222,6 +222,14 @@
     <value>Accounts:</value>
     <comment>This is the header for the list of accounts.</comment>
   </data>
+  <data name="Message.AppRestored" xml:space="preserve">
+    <value>The application has been restored successfully.</value>
```

Comment:
Could you please change to Eppie account restored?

Reply:
done at [commit](https://github.com/Eppie-io/Eppie-CLI/pull/82/commits/392e5e3a11abf4947162a6a833ac17e0319efa6a)